src/hostManager/hostManager.ts

```diff
@@ -68,14 +68,19 @@ export class ClientHostManager {
     return { host: new URL(host), prefix };
   }
 
-  public getItemAsURL(context: Context, itemId: string) {
+  public getItemAsURL(context: Context, itemId: string, qs: object = {}) {
```

Comment:
I am not sure we should use `object` type. Can you use `{ [key: string]: string | number | boolean }` ?